sdlf-ddk-lightweight/app.py

```diff
@@ -13,72 +13,98 @@
 # limitations under the License.
 
 
-from typing import Any, Dict
-import wrapt
+from typing import Any, Tuple, Dict
+
 import aws_cdk as cdk
+import wrapt
+from aws_cdk.pipelines import CodePipeline
 from aws_ddk_core.cicd import CICDPipelineStack
-from data_lake.pipelines import SDLFPipelineStack
 from aws_ddk_core.config import Config
-from aws_cdk.pipelines import CodePipeline
+from constructs import Construct
+
+from data_lake.pipelines import SDLFBaseStack
 
 
-@wrapt.patch_function_wrapper(CICDPipelineStack, 'build')
-def build(wrapped, self, args, kwargs) -> "CICDPipelineStack":
+@wrapt.patch_function_wrapper(CICDPipelineStack, 'build')  # type: ignore
```

Comment:
Did you consider contributing this to main [ddk](https://github.com/awslabs/aws-ddk) repo?

Reply:
Can open up an issue for this - using wrapt was our workaround here due to the limited number of file assets that AWS CodeBuild can publish concurrently (otherwise typically stuck retrying CodeBuild stage as each asset gets publishes 1 by 1 for new AWS accounts)

Reply:
Issue: https://github.com/awslabs/aws-ddk/issues/182

Reply:
Awesome, thanks!